src/containers/registrationform.js

```diff
@@ -0,0 +1,162 @@
+import React from 'react';
+import './style.css';
+import jquery  from "jquery";
+
+
+class RegisterForm extends React.Component {
+    constructor() {
+        super();
+        this.state = {
+            fields: {},
+            errors: {}
+        }
+
+        this.handleChange = this.handleChange.bind(this);
+        this.submituserRegistrationForm = this.submituserRegistrationForm.bind(this);
```

Comment:
have to reduce a handlechange

Reply:
Try arrow function to avoid this binding.